Manifests/ManagedPreferencesApplications/xyz.techitout.appAutoPatch.plist

```diff
@@ -1141,6 +1158,44 @@ During a profile replacement, the system updates payloads with the same `Payload
 			<key>pfm_type</key>
 			<string>string</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>3.5.0</string>
+			<key>pfm_description</key>
+			<string>Determines the version comparison method used when checking for updates</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/App-Auto-Patch/App-Auto-Patch/wiki/Version-Comparison-Methods#version-comparison-method</string>
+			<key>pfm_name</key>
+			<string>versionComparisonMethod</string>
+			<key>pfm_range_list</key>
+			<array>
+				<string>IS_AT_LEAST</string>
+				<string>EQUAL_TO</string>
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>Is At Least</string>
+				<string>Equal To</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Version Comparison Method</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>3.5.0</string>
+			<key>pfm_description</key>
```

Comment:
Looks like the default behavior is `false`. Could you please add a `pfm_default` indicating that?